packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -967,6 +968,10 @@ class Scaffold extends StatefulWidget {
   /// Typically a [Drawer].
   final Widget endDrawer;
 
+  /// Set a color to use for the scrim that obscures primary content
+  /// while a drawer is open.
+  final Color scrimColor;
```

Comment:
Since this is only used for the drawer, I suggest renaming to `drawerScrimColor`.

Reply:
done